doc/generic/pgf/pgfmanual-en-pgfkeysfiltered.tex

```diff
@@ -74,21 +80,21 @@ \subsubsection{Starting With An Example}
 %
 \noindent and
 %
-\begin{codeexample}[code only]
+\begin{codeexample}[code only,preamble={\usepgfkeyslibrary{filtered}}]
```

Comment:
Since `filtered` is loaded by default, there seems no need to add `preamble={\usepgfkeyslibrary{filtered}}`?

Reply:
In the future I want to remove the default loading.

Reply:
Then in a more strict manner `preamble={\usepgfkeyslibrary{filtered}}` should be added when `filtered` is not loaded by default. From another aspect, tikz library `topaths` is loaded by default and all its `codeexamples` don't have `preamble={\usetikzlibrary{topaths}}`.

If the removal of default loading will happen soon (which requires a new release of `pgfplots`), then I'm OK with this temporary redundancy.